### PR TITLE
feat: Extend multiple sections type of escapeAssertion

### PR DESCRIPTION
**Pull request by Gabriel-403**
At present, only p < int > is supported. We should consider more forms. For example, consider supporting P < string > to meet the following test cases
r.attr.value == p.attr
r.attp.value || p.attr
r.attp.value && p.attr
r.attp.value >p.attr
r.attp.value <p.attr
r.attp.value +p.attr
r.attp.value -p.attr
r.attp.value *p.attr
r.attp.value /p.attr
!r.attp.value /p.attr'
g(r.sub, p.sub) == p.attr
g(r.sub,p.sub) == p.attr
(r.attp.value || p.attr)p.u

### CODE REVIEW

File: test/util.test.ts [hunk 1]
```diff
@@ -190,3 +190,19 @@ test('test keyGet2Func', () => {
...
+  expect(util.escapeAssertion('r.attp.value >p.attr')).toEqual('r_attp.value >p_attr');
+  expect(util.escapeAssertion('r.attp.value <p.attr')).toEqual('r_attp.value <p_attr');
+  expect(util.escapeAssertion('r.attp.value +p.attr')).toEqual('r_attp.value +p_attr');
+  expect(util.escapeAssertion('r.attp.value -p.attr')).toEqual('r_attp.value -p_attr');
+  expect(util.escapeAssertion('r.attp.value *p.attr')).toEqual('r_attp.value *p_attr');
+  expect(util.escapeAssertion('r.attp.value /p.attr')).toEqual('r_attp.value /p_attr');
+  expect(util.escapeAssertion('!r.attp.value /p.attr')).toEqual('!r_attp.value /p_attr');
+  expect(util.escapeAssertion('g(r.sub, p.sub) == p.attr')).toEqual('g(r_sub, p_sub) == p_attr');
+  expect(util.escapeAssertion('g(r.sub,p.sub) == p.attr')).toEqual('g(r_sub,p_sub) == p_attr');
+  expect(util.escapeAssertion('(r.attp.value || p.attr)p.u')).toEqual('(r_attp.value || p_attr)p_u');
```
Comment (nodece) [inline test/util.test.ts:207]: could you add more cases, so like:
```
r_app1.value == p.value
r_app1.value == p_app1.value
rapp1.value == p.value
rapp1.value == papp1.value
```

Comment (Gabriel-403) [reply test/util.test.ts:207]: This matching method will be supported in the future！

Comment (nodece) [reply test/util.test.ts:207]: You are solving is this problem, why should you support this in the future?

Comment (sagilio) [reply test/util.test.ts:207]: @nodece It looks like we can not read the policy type like `r_app1`? `sec + Model.getKeySuffix(i)` will return like `r1` `r2` and so on.
https://github.com/casbin/casbin.js/blob/72a658b43c7037b4e7ae4c956168c578e535d17c/src/model/model.ts#L64-L73

Comment (nodece) [reply test/util.test.ts:207]: @sagilio thanks for your point.

Comment (nodece) [reply test/util.test.ts:207]: Hi @Gabriel-403, thanks for your contribution, casbin cannot support the policy type like `r_app1`, so you need to reset `escapeAssertion` method, and append the `r1.value == p1.value` and `r2.value == p4.value` to test cases.

**Comment by nodece** [comment]
LGTM

**Review by nodece — APPROVED** [review]

**Review by sagilio — APPROVED** [review]

**Pull request merged**